pkg/scheduler/listers.go

```diff
@@ -52,3 +54,42 @@ func (f FakePodLister) ListPods(s labels.Selector) (selected []api.Pod, err erro
 	}
 	return selected, nil
 }
+
+// ServiceLister interface represents anything that can produce a list of services; the list is consumed by a scheduler.
+type ServiceLister interface {
+	// Lists all the services
+	ListServices() (api.ServiceList, error)
+	// Gets the services for the given pod
+	GetPodServices(api.Pod) ([]api.Service, error)
+}
+
+// FakeServiceLister implements ServiceLister on []api.Service for test purposes.
+type FakeServiceLister []api.Service
+
+// FakeServiceLister returns api.ServiceList, the list of all services.
+func (f FakeServiceLister) ListServices() (api.ServiceList, error) {
+	return api.ServiceList{Items: f}, nil
+}
+
+// GetPodService gets the service that has the selector that can match the labels on the given pod
+// We are assuming a single service per pod.
+// In case of multiple services per pod, the first service found is returned
```

Comment:
That's fine, but is there something I need to do to view the new commit? When I view this on Github it still says "In case of multiple services per pod, the first service found is returned"